Index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <nav id="navbar">
+        <header>Technical documentation</header>
+        <ul>
+            <li><a class="nav-link" href="Introduction">Introduction</a></li>
+            <li><a class="nav-link" href="What_you_should_know">What you should know</a></li>
+            <li><a class="nav-link" href="While_Statement">While Statement</a></li>
+            <li><a class="nav-link" href="Hello_World">Hello World</a></li>
+            <li><a class="nav-link" href="Variables">Variables</a></li>
```

Comment:
Kindly use links in your <href> to ensure that your links are working.

Reply:
- [ ] The same here be specific on what you want , give the reason why to use link, provide documents in where they how to connect links in the project :wink:  :upside_down_face: